server/db_ql_test.go

```diff
@@ -58,6 +58,21 @@ func TestQlFixity(t *testing.T) {
 			if id != tab.id {
 				t.Errorf("Received %s, expected %s", id, tab.id)
 			}
+		case "PostAndPutFixity":
```

Comment:
Are there also similar tests for the MySQL adapter?

I'm thinking that most of these tests could probably apply to both adapters. Unifying it could be a future project.